src/static/helpers/slasHelper.ts

```diff
@@ -110,21 +110,23 @@ export const generateCodeChallenge = async (
  * @param privateClient? - flag to indicate if the client is private or not. Defaults to false.
  * @returns login url, user id and authorization code if available
  */
-export async function authorize(
+export async function authorize(options: {
   slasClient: ShopperLogin<{
     shortCode: string;
     organizationId: string;
     clientId: string;
     siteId: string;
-  }>,
-  codeVerifier: string,
+  }>;
```

Comment:
how is this code working, we are changing from colon to semincolon. The colon is to separate the arg in the same function, while the semicoln does not really make sense